editions/tw5.com/tiddlers/filters/debug-log.tid

```diff
@@ -0,0 +1,13 @@
+caption: debug-log
+created: 20230111163645018
+modified: 20230111163645018
+op-input: a [[selection of titles|Title Selection]]
+op-output: the original input titles
+op-parameter: name of a variable
+op-parameter-name: V
+op-purpose: prints a table of the input titles to the console; if optional parameter <<.place V>> is specified, also prints the value of the variable with that name
+tags: [[Conditional Operators]] [[Filter Operators]]
+title: debug-log Operator
+type: text/vnd.tiddlywiki
+
+<<.from-version "5.2.6">>
```

Comment:
IMO there should be some more text, that the `debug-log` operator should be removed from the working filter expression. Otherwise it will only slow down the execution and pollute the dev console

Reply:
I have added a longer explanation of the filter's purpose that leans on the description of $action-log. I hope it's not too verbose.
It also contains a reminder to clean up.